docs/pages/application-access/cloud-apis/azure-aks-workload-id.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: "Protect Azure CLIs with Teleport Application Access on AKS"
+description: How to enable secure access to Azure CLIs on Azure Kubernetes Service with Workload Identity.
+---
+
+(!docs/pages/includes/application-access/azure-intro.mdx!)
```

Comment:
The intro paragraph could use a smoother transition to the numbered list. We go from a description of the ability to protect CLI tools with Teleport to a list of objectives in the guide:

<img width="731" alt="image" src="https://github.com/gravitational/teleport/assets/25250668/caec0caf-d14c-4e46-8251-46a7f181d69b">